sound/soc/sof/imx/imx8.c

```diff
@@ -39,6 +39,8 @@
 #define MBOX_OFFSET	0x800000
 #define MBOX_SIZE	0x1000
 
+MODULE_IMPORT_NS(SND_SOC_SOF_XTENSA);
+
```

Comment:
usually this is added close to the MODULE_LICENSE, e.g. line 480. Adding this in the middle of the code is odd.